var/spack/repos/builtin/packages/r-dt/package.py

```diff
@@ -25,3 +28,4 @@ class RDt(RPackage):
     depends_on('r-htmlwidgets@1.3:', type=('build', 'run'))
```

Comment:
Missing dependency:

> jsonlite (≥ 0.9.16)

Reply:
Another case where `jsonlite` moved from "suggests" to "imports" in an update that I did after checking dependencies. Added the constraint when version is 0.8 and above.